frontend/app/model/ResponsiveImage.scala

```diff
@@ -7,7 +7,7 @@ case class ResponsiveImageGroup(
   altText: String,
   availableImages: Seq[ResponsiveImage]
 ) {
-  val defaultImage = availableImages.head.path
+  val defaultImage = availableImages.headOption.map(_.path)
```

Comment:
@feedmypixel We shouldn't be returning a `ResponsiveImageGroup` if there are no available image sizes, so I think this is OK to no be an option.